ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -446,6 +470,10 @@ object PackageService {
   ): b.Resolved =
     b(pkgId, qn.module.dottedName, qn.name.dottedName)
 
+  private def getPackageLanguageMap(packageStore: PackageStore): Map[String, LanguageVersion] = {
+    packageStore.view.map({ case (p, s) => p -> s.languageVersion }).toMap
```

Comment:
it could be even more concise without the view and toMap, since it's only one transformation stage